src/Microsoft.ML.Data/Prediction/IPredictionTransformer.cs

```diff
@@ -2,22 +2,23 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.Internal.Calibration;
-using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace Microsoft.ML.Runtime
 {
     public interface IPredictionTransformer<out TModel> : ITransformer
         where TModel : IPredictor
+    {
+        TModel Model { get; }
+    }
+
+    public interface IClassicPredictionTransformer<out TModel> : IPredictionTransformer<TModel>
```

Comment:
>Classic [](start = 22, length = 7)

'Classic' sounds a bit wacky, even though it was my suggestion. Maybe 'SingleInput' ?..

Reply:
Classic is weird.

---
In reply to: [218633462](https://github.com/dotnet/machinelearning/pull/912#discussion_r218633462) [](ancestors = 218633462)